application/common/models/MfaWebauthn.php

```diff
@@ -92,4 +92,17 @@ public static function createWebauthn(Mfa $mfa, string $keyHandleHash, string $l
 
         return $webauthn;
     }
+
+    public function setLastUsed()
+    {
+        $this->last_used_utc = MySqlDateTime::now();
+        if (! $this->save()) {
```

Comment:
This saves more than just last_used_utc. You could limit it to just last_used_utc. But this works.

Reply:
What's the syntax for that? I could look it up, but I'm sure you can whip it right out.

Reply:
$this->save(true, ['last_used_utc']) -- still runs the validation, but only validate and save last_used_utc.

Reply:
Thank you very much.